samples/nrf9160/secure_services/src/main.c

```diff
@@ -88,16 +88,30 @@ void main(void)
 #endif
 
 	u32_t info_part = 0;
-	u32_t ficr_addr = (u32_t)&NRF_FICR_S->INFO.PART;
+	u32_t ficr_info_part_addr = (u32_t)&NRF_FICR_S->INFO.PART;
 
-	printk("\nRead FICR, offset 0x20C (address 0x%08x):\n", ficr_addr);
-	ret = spm_request_read(&info_part, ficr_addr, sizeof(u32_t));
+	printk("\nRead FICR, offset 0x20C (address 0x%08x):\n",
```

Comment:
These two calls look repetitive, would it make sense to extract a function for this?

```
ficr_read(u32_t addr, char *offset, char *name)
```

Reply:
Done